content/agent/kubernetes/host_setup.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Kubernetes Host Setup
+kind: documentation
+further_reading:
+- link: "agent/autodiscovery"
+  tag: "documentation"
+  text: Docker Agent Autodiscovery
+- link: "agent/kubernetes/host_setup"
+  tag: "documentation"
+  text: "Kubernetes Host Setup"
+- link: "agent/kubernetes/integrations"
+  tag: "documentation"
+  text: "Custom Integrations"
+---
+
+Installing the Agent directly on your host (rather than having the Agent run in a Pod) provides additional visibility into your ecosystem, independent of Kubernetes.
```

Comment:
Are we talking of the package install on the host, or having the container launched on the host rather than in a pod?

On the host install there's a bunch of additional things to configure:
* kubelet configuration in the agent config (which is done by env var in the image)
* activating the kubelet listener & config provider in the agent config
* enabling the kubelet (and optionally apiserver) check configurations